### DISCUSSION TITLE
Disabling turret ammo when ammo system is turned off?

### DISCUSSION THREAD

**Discussion by MisterBiter** [CombatExtended-Continued/CombatExtended#3547] [General]
Hey guys, I'm not sure if it is a bug, probably is. But when you play the game without ammo system enabled then the turrets only shoot until their "magazine" runs empty. 
Would make sense to disable turret magazine I guess when disabling ammo system from options. 

Is there a way to do it?

**Comment by perkinslr** [comment]
There is an obvious bug currently, where the mech auto-mortar needs a mech to reload it every few shots.  I think it is intended that turrets (manned or otherwise) should require reloading, but auto turrets should *probably* auto reload.  This would cause a pause in their shooting, but not require a pawn to get involved.  Especially since the ammo remaining is not displayed.  Otherwise, the remaining ammo *should* be displayed, so the player knows when they need to order a reload.

Anyway, that's probably something to bring up in #balancing on discord.  Since it isn't completely broken, *I*'m not going to prioritize fixing it, but I'll happily review a PR implementing either solution.  If no one gets to it before my next bug hunt, I'll probably re-enable the ammo-remaining gizmo on turrets.
